src/sidebar/store/modules/test/annotations-test.js

```diff
@@ -147,14 +168,56 @@ describe('annotations reducer', function() {
       const ann = fixtures.defaultAnnotation();
       store.dispatch(actions.addAnnotations([ann]));
       store.dispatch(actions.removeAnnotations([ann]));
-      assert.equal(store.getState().annotations.length, 0);
+      assert.equal(store.getRootState().annotations.annotations.length, 0);
     });
   });
 
   describe('#updateFlagStatus', function() {
-    unroll(
-      'updates the flagged status of an annotation',
-      function(testCase) {
+    [
+      {
+        // Non-moderator flags annotation
+        wasFlagged: false,
+        nowFlagged: true,
+        oldModeration: undefined,
+        newModeration: undefined,
+      },
+      {
+        // Non-moderator un-flags annotation
+        wasFlagged: true,
+        nowFlagged: false,
+        oldModeration: undefined,
+        newModeration: undefined,
+      },
+      {
+        // Moderator un-flags an already unflagged annotation
+        wasFlagged: false,
+        nowFlagged: false,
+        oldModeration: { flagCount: 1 },
+        newModeration: { flagCount: 1 },
+      },
+      {
+        // Moderator flags an already flagged annotation
+        wasFlagged: true,
+        nowFlagged: true,
+        oldModeration: { flagCount: 1 },
+        newModeration: { flagCount: 1 },
+      },
+      {
+        // Moderator flags annotation
+        wasFlagged: false,
+        nowFlagged: true,
+        oldModeration: { flagCount: 0 },
+        newModeration: { flagCount: 1 },
+      },
+      {
+        // Moderator un-flags annotation
+        wasFlagged: true,
+        nowFlagged: false,
+        oldModeration: { flagCount: 1 },
+        newModeration: { flagCount: 0 },
+      },
+    ].forEach(testCase => {
+      it('updates the flagged status of an annotation', () => {
```

Comment:
I realize this description was taken from existing code. I think it would be helpful here to add some discriminator to the description to indicate which case failed, in case any of these tests fail. I typically do this by putting something in brackets at the end of the string (``it(`does something ${testCase.description}`)``).